src/harness/util.py

```diff
@@ -125,3 +135,14 @@ def makePpaAndPalRecordsConsistent(ppa_record, pal_records, low_frequency,
   # Converting from defaultdict to dict
   ppa_record = json.loads(json.dumps(ppa_record))
   return ppa_record, pal_records
+
+
+def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(_sas_admin):
```

Comment:
It could also be in sas_testcase.py so you don't have to pass in the SAS admin instance (I think it'll be less prone to error). If you keep it here, the param name should probably change to "sas_admin" to match Python naming conventions.